### ISSUE TITLE
21/30 Spawntimes

### ISSUE THREAD

**Issue by msh100** [msh100/ETPro-configs#2]
Change spawntimes on 21/30. This needs more discussion.

**Comment by Sebhes** [comment]
Supply definitely needs a change in spawntime as teams sometimes have a too huge advantage of a "bad" spawntime. Having 21/30 sec spawns would fix this. However, I don't know how this will work out on other maps. 

In goldrush you can also get stuck on the last stage if you have a bad spawntime. How do you think this will work out on other maps, besides grush/supply?

**Comment by msh100** [comment]
I made a branch with these changes. We should test and get feedback on them.

https://github.com/msh100/ETPro-configs/compare/supply-gr-spawntime-proposals

**Comment by Swanidius** [comment]
Supply is good but u edited wrong map. Goldrush instead of sw_goldrush

**Comment by msh100** [comment]
Ah nice spot, I have added it to `sw_goldrush_te`.

**Comment by Swanidius** [comment]
all good now

**Comment by ohurcool** [comment]
Too bold of a change at this point in the game's life. If this was such an issue, why wasn't it changed years ago? Think of how many players will be upset about this - forcing everyone (in both offis and IRC wars, since many server owners will immediately adopt the new config) to adapt to an unnecessary gameplay change which only a few players support, most of which are either friends with each other / in the same team as each other or inactive. Whoever was the "face" of this change would be flamed hard.

Besides, I think the effect spawntimes have on gameplay is blown out of proportion. It's very possible to quickly advance past any stage regardless of who spawns when. The problem is that many teams play the same way every spawn no matter what. Compare it to switching between CP spawn and flag spawn when attacking on radar or choosing to push very hard in the first few seconds of the round due to a "short spawn", or abandoning a CP build in order to complete a later objective when one or two defending players are on full... the list goes on. You should attack based on all sorts of variables, and spawntimes are one of them. Plus, it's been said before that the team who executes better will always win on supply and goldrush.

I do not support this as both a player and an admin. That being said, here is something to consider: http://www.crossfire.nu/threads/47471/combined-fixes I know of a few exploits which the current global server configs do not fix.

Changing the FPS limit is fine in my opinion. Don't remove prone delay or change the timelimits of maps, as I've seen suggested on CF.

**Comment by Sebhes** [comment]
I would like to react on ohurcool.

I do agree that making big changes as changing the spawntime is a difficult topic. Especially, in ET 2015. I would like to point out that my opinion is one from the most active ET players these days. Since January 2014, I have been the most active player in the ET 3on3 scene. I have gained a lot of experience throughout this period and from what I know is that there does exist something like a "bad" spawntime. Atleast on the maps supply and sw_goldrush_te. If the attack team has a bad spawntime, the defending team has "free" pushes without much risk. These free pushes will weaken the attacking team in some way, for example ammo-, time- wise or health wise. The spawntimes are random and it is good possible that the next round this bad spawntime will not excist. Therefore, one team has unfairly gained a huge advantage.

I should also point out that these bad spawntimes only occur versus equally skilled opponents. I find it save to state that I have lost important games due this spawntime issue, which could have been avoided with 21/30. Having a bad spawntime gives my team a significant disadvantage. 

I am a big fan of changing the spawntimes. However, the average ET player will most likely not even be aware of this issue and therefore I find it difficult to state whether I am pro or con.

In short, there is a certain advantage for teams in 3on3 who are equally skilled in the atleast the maps sw_goldrush_te and supplydepot which could possible be avoided with 21/30-spawntimes. I have not enough experience to answer the same question in 6on6. 

This having said, I don't know if the majority will agree with me and therefore I don't know whether we should implement this feature.

**Comment by Swanidius** [comment]
Im not gonna post wall of text so here is my short opinion.

I agree totally what sebhes said. If we are not gonna use this in 6v6 we definitely need this for 3v3. Like its a must. But only for Supply and sw_goldrush.

and my opinion is from a guy who has played this game actively since the dawn of ET and I know how frustrating is to play vs perfect spawntime.

**Comment by msh100** [comment]
So it sounds like only changing this on `global3` is the right way to go. I think we need substantially more input however. This should be opened up for discussion on crossfire.

**Comment by Swanidius** [comment]
agree

**Comment by ohurcool** [comment]
I still disagree with the change but won't bother commenting any further on it. You should definitely post a journal or something on CF beforehand. :P

**Comment by Artztar** [comment]
"The problem is that many teams play the same way every spawn no matter what."

^ this. I've seen amazing spawntimes still being beaten by smarter gameplay, it is harder of course, but that's the entire point. if you don't know how to avoid being spawnkilled then you're playing the wrong game. I may not have been extremely active in the last year or more but in my most active years I experienced and witnessed these so called "perfect spawntimes" being outplayed by teams who understand patient and passive gameplay as much as they do the will to push instantly after spawning. I disagree completely with the 21/30 change.

**Comment by Sebhes** [comment]
I am not sure what these different ways of attacking have to do with spawntimes. In my opinion it is about creating equal circumstances for both teams. I am not the kind of player myself who pushes each spawn in exact the same manor and I can perfectly adapt myself to different circumstances. However, my point is that bad spawntimes do give a significant  disadvantage to the attacking team. With 21/30 you could possibly avoid this.

To clarify, it is not about lacking adaptability, it is about creating equal playing field.

**Comment by Sebhes** [comment]
I don't see this feature being implemented because there is simply to few support for it. It is a pity, because in my opinion this would have made the game so much more attractive.

**Comment by msh100** [comment]
The way I see it is there will be equal playing no matter what these values are set to. Both teams ultimately play with both spawn times.

Further to that by changing the spawn times you are likely to introduce other quirks that didn't exist in the old times. I don't think I can agree that changing times 'fixed' absolutely anything.

Sent from my iPhone

On 26 May 2015, at 13:30, Sebhes <notifications@github.com<mailto:notifications@github.com>> wrote:

I am not sure what these different ways of attacking have to do with spawntimes. In my opinion it is about to create an equal playing field for both teams. I am not the kind of player myself who pushes each spawn in exact the same manor and I can perfectly adapt myself to different circumstances. However, my point is that bad spawntimes do give a significant disadvantage to the attacking team. With 21/30 you could possibly avoid this.

To clarify, it is not about lacking adaptability, it is about creating equal circumstances.

##

**Comment by Sebhes** [comment]
@msh100, I complete disagree with your statement: "The way I see it is there will be equal playing no matter what these values are set to. Both teams ultimately play with both spawn times."

The entire point of having 21/30 is to avoid having unequal circumstances over 2 rounds. 
For example, team A can have a "bad" spawntime while attacking. After round 1 has finished, team B doesn't necessarily need to have  a "bad" spawntime theirselves. This and this entire point has just made the map unfair for team A. 

This is beyond the scope of how teams should attack. The fact is that the defending team (having a "good" spawntime) can push out more often before their spawntime without taking any risk, forcing the attacking team to hide in some way or avoid getting killed, atleast wasting some time. 

Yesterday I played myself the map adlernest versus Sungi, Lordi and Rong and we had a perfect spawntime while defending. I am pretty sure this gave us a significant advantage in setting up and pushing out without taking any risk.

**Comment by Artztar** [comment]
"I am not sure what these different ways of attacking have to do with spawntimes."

I can't say I'm feeling energetic enough to write a tutorial on how to hide for the ~5-10 seconds of push time axis would have on the allied team. So I disagree, spawntimes are completely related to how passive or aggressive the attacking or defending teams should play. This is what ET is. It's the way it's always been. This is so complex that it doesn't just start and stop at 'axis have the advantage, unfair to allies.'

**Comment by Sebhes** [comment]
I would like to have the opportunity to have a match config where the spawntimes are 21/30. I think it would make the game more dynamic and less predictable. It doesn't need to be the config by default, but having a config exhibiting this feature would be nice.

**Comment by ohurcool** [comment]
ok so I think it's clear we aren't changing the spawntimes, so can we consider releasing the new configset soon maybe? :P

**Comment by ohurcool** [comment]
bumpity bump

**Comment by Swanidius** [comment]
msh100 where are u bro!

**Comment by Sebhes** [comment]
Msh100 !

**Comment by miraya** [comment]
The idea for changing spawntime came from the obsessive need to stop double fullholds on supply due to excessive favored spawntime of Axis, this however in my opinion has absolutely no positive impact on the current state of the game. Why so may you ask, this request has been inquired many times before but hardly half of the community agrees on this. To make a major change such as this it would be better to have some feedback from the community either through a voting system/poll and ask for serious input why the change should go through or not at all. Honestly if spawntimes had this much of a huge impact on the game I think somewhere you have adopted the wrong concept of how to play ET, spawntimes do have an impact but that impact is minimalism depending on the player. Your playstyle defines how much spawntime has an impact on you not being able to advance forward in a map stage.

TL;DR: Not really adoptable, ask community for feedback, spawntimes depend hugely on the player aswell.

**Comment by msh100** [comment]
Sounds like you have the same views as the rest of us. Overall this doesn't seem like a positive change.

**Comment by ohurcool** [comment]
lalalala

**Comment by solidsneak** [comment]
I think it's been blown out of proportion by players who use it as an excuse for a loss and it's fine the way it is. Also, to say "this is the way it's always been" isn't a good argument but a fallacy.
